prepare-docs.rb

```diff
@@ -11,6 +11,7 @@
 $docs_repo_root = $docs_repo + "/blob/main"
 $docs_repo_branch = "final"
 $addons_repo_branch = "main"
+$addons_repo_branch_outdated = "master" # Fix edit links for bundles with iown repositories.
```

Comment:
Why "outdated"? :)

Reply:
I have switched to main (which is prbably the generic one for future repos) for the docs and addons repo to fix all other links.
I would except the other repos to switch to `main` at some time in future too, thats why i called it outdated. :)